package.json

```diff
@@ -71,5 +71,9 @@
 	"engines": {
 		"yarn": "^1.0",
 		"node": "^10.4.1"
+	},
+	"dependencies": {
```

Comment:
I don't think these should be added at the top level project.json, will it work without it?